quiz.py

```diff
@@ -1,6 +1,54 @@
 """
 File: quiz.py
-Author: Name
-Date: YYYY-MM-DD
-Description: A brief explanation of what this program does.
-"""
\ No newline at end of file
+Author: Jero Cansanay
+Date: 2024-03-05
+Description: A Multiple choice game that calculates the score
+"""
+print("Multiple-Question Quiz Game (NBA)")
+
+correct_answers = 0
+
+print("1. Who scored the most career points in the NBA?")
+print("(a) Michael Jordan")
+print("(b) Lebron James")
+print("(c) Kareem Abdul-Jabbar")
+
+answer_1 = input("Answer:")
```

Comment:
I see you're using separate variables to track user input. It doesn't appear that you do anything with the three variables at the end of the program — in other words, why do you need to remember what the user entered for questions 1, 2, 3, etc.? You can make the program more memory efficient by using only one variable for user input, then reassigning (reusing) it every new question.